src/js/scroll.js

```diff
@@ -14,10 +13,10 @@ export function initScroll() {
     scrollOffset = 40;
   }
 
-  debounce(smoothScroll.init({
+  smoothScroll.init({
```

Comment:
I can put this inside a `requestIdleCallback` but I don't think it's such a perf bottleneck.